### PR TITLE
Refactor model definition of `examples/pytorch_simple.py`

### PR DESCRIPTION
**Pull request by crcrpar**
By using `nn.Sequential` that automatically defines its forward computation from the arguments of `__init__`, the code gets shorter and less redundant in my opinion.

### CODE REVIEW

File: examples/pytorch_simple.py [hunk 1]
```diff
@@ -40,40 +40,24 @@
...
 
 
-class Net(nn.Module):
-    # Constructor for trial network.
-    def __init__(self, trial):
-        super(Net, self).__init__()
-        self.layers = []
-        self.dropouts = []
-
-        # We optimize the number of layers, hidden untis in each layer and drouputs.
```
Comment (toshihikoyanase) [inline examples/pytorch_simple.py]: This comment is still relevant to the new code, and I think it is helpful for novice users of Optuna.
So, how about adding this comment to L44?

Comment (crcrpar) [reply examples/pytorch_simple.py]: That sounds great!

**Comment by crcrpar** [comment]
[nit] Just a comment. I noticed that the original implementation only samples dropout probability once even if the number of layers is > 1 while this PR optimizes dropout ratio for each layer. In addition to this, other examples like `chainer_simple.py` and `tensorflow_eager_simple.py` don't optimize it.

**Comment by toshihikoyanase** [comment]
> this PR optimizes dropout ratio for each layer
> other examples like chainer_simple.py and tensorflow_eager_simple.py don't optimize it.

Thank you for pointing it out.
I think we do not have specific reasons to apply dropout only to the PyTorch example. It may be helpful if all examples of deep learning frameworks have the same search space (i.e., hyperparameters). Anyway, we can separate the PRs, and I'll approve and merge this PR.
